src/CloudSecrets.UnitTests/SecretsAnalyzerTests.cs

```diff
@@ -40,7 +40,7 @@ public class SecretsAnalyzerTests
         private const string ValidFilePath = "c:\\test";
         private const string ValidFileContent = "this is a content";
 
-        [TestMethod, Ignore("Secrets not yet supported")]
+        [TestMethod, Ignore("Secrets not yet supported")] // __secrets
```

Comment:
I haven't deleted anything; instead, I've comment bits out, and tagged them all with `__secrets` (including the MEF-changes you made earlier today).